lib/client-assets.php

```diff
@@ -440,6 +440,19 @@ function gutenberg_register_packages_styles( $styles ) {
 }
 add_action( 'wp_default_styles', 'gutenberg_register_packages_styles' );
 
+/**
+ * Gets the stylesheet handle for the current theme.
+ *
+ * @since 6.3
+ *
+ * @return string
+ */
+function gutenberg_get_stylesheet_handle() {
+	$stylesheet_array = explode( '/', get_option( 'stylesheet' ) );
+	$stylesheet_slug  = end( $stylesheet_array );
+	return $stylesheet_slug . '-style';
```

Comment:
It seems like themes can register their theme styles as whatever slug they want, so I don't think we can guarantee this naming convention to get the slug of the stylesheet. Maybe there's another action that reliably fires right before scripts are output, such as `[print_styles_array](https://developer.wordpress.org/reference/hooks/print_styles_array/)`, and then we can manually add it to the end of the styles array? I'm not sure if that would work though.